Ci 235

andrewbird opened the pull request:
Here's the simple serial tests, fossil will come later in another PR (I found a 3 year old branch where I started to write some already)

andrewbird committed:
CI: Avoid printing wrapper in failure traceback

andrewbird committed:
CI: Add simple serial test read from echo

andrewbird committed:
CI: Add simple serial test write to file

stsp commented:
> Here's the simple serial tests, fossil will come later in another PR

You may also consider
similar simple LPT tests
in that another PR. :)

andrewbird commented:
Actually, I'll add it to this one, it's likely to be quicker. What $_lpt settings should I use? I had a quick look at the dosemu.conf and the examples show direct to /dev/lpt, or presumably via a pipe to linux print command. Would a `cat - > file` be okay?

stsp commented:
Yes, cat should be fine.

andrewbird committed:
CI: Add simple lpt test write to pipe

LPT devices are connected either to a device file e.g. '/dev/lpt1'
or piped to a command sich as 'lpr -P PDF'.

stsp commented:
Thanks!
Very helpful.
